src/content/deadlines/04_photos.mdx

```diff
@@ -1,8 +1,10 @@
 ---
-title: Photos
-subtitle: Refresh you memories!
-url: /photos
+title: 2024 Videos
+subtitle: Upgrade your experience
+url: https://www.youtube.com/playlist?list=
 image: "./enjoy.jpg"
 ---
 
-Remember the days at the conference? Have a look at our photo gallery to recall all those nice memories.
+Any sessions you wanted to attend but could not due to too many parallel tracks? Or maybe you missed a session because you were too tired or had to take care of something else?
+
+Go to our playlist and watch 3 days of EuroPython 2024 sessions. Including all keynotes, talks, and lightning talks.
```

Comment:
To me, this is way too wordy for CTA. Maybe something like "Missed a session or couldn’t make it to the conference? Dive into the recordings and immerse yourself in 3 days of inspiring talks, keynotes, and lightning talks. Learn together and relish in our shared love for Python!" (You can even omit the first sentence)
I think it is generally good not to have multiple paragraphs for the CTA.